docs/source/markdown/podman-system-connection.1.md

```diff
@@ -11,6 +11,17 @@ Manage the destination(s) for Podman service(s).
 
 The user is prompted for the ssh login password or key file passphrase as required. The `ssh-agent` is supported if it is running.
 
+Podman manages the system connection by writing and reading the `podman-connections.json` file located under
```

Comment:
These should probably be handled via the options subdir, to make sure they don't drift.

Reply:
AFAIK the options syntax is for cli options cannot be used here.

Reply:
@edsantiago Any reason this would not work?